pkgs/top-level/all-packages.nix

```diff
@@ -37364,6 +37364,8 @@ with pkgs;
 
   ### GAMES/DOOM-PORTS
 
+  dsda-doom = callPackage ../games/doom-ports/dsda-doom { };
```

Comment:
Use [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name) to avoid editing this file